.gitpod.Dockerfile

```diff
@@ -21,7 +21,7 @@ ENV PATH="/home/gitpod/.nvm/versions/node/v${NODE_VERSION}/bin:${PATH}"
 
 RUN curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s
 # Wasp gets installed in $HOME/.local/bin, so we need to add it to PATH.
-RUN printf 'export PATH="%s:$PATH"\n' "$HOME/.local/bin" >> $HOME/.bashrc
+ENV PATH="/home/gitpod/.local/bin:${PATH}"
```

Comment:
Nice, simplification!
Aha, but this also makes sure that home works correctly, ok makes sense. I believe that is because VSCode doesn't expand `$HOME`, or smth in that direction.